readme.md

```diff
@@ -1,51 +1,65 @@
 
 #### CLI
+
 Interacting with forklift is done via the [command line interface](src/forklift/cli.py). Run `forklift -h` for a list of all of the available commands.
 
 #### Config File Properties
+
 `config.json` is created in the working directory after running `forklift config init`. It contains the following properties:
-- `warehouse` The folder where all of the repositories will be cloned into and where forklift will scan for pallets to run.
-- `sendEmails` Determines whether or not to send any emails. Set to `false` when testing.
-- `notify` A list of emails that will be sent the summary report each time `forklift lift` is run (assuming `sendEmails: true`).
-- `repositories` A list of repositories (`<owner>/<name>`) that will be cloned/updated into the warehouse folder.
-- `copyDestinations` A list of folders that you want any data defined in `Pallet.copy_data` to be copied to upon successful processing of the pallet.
+
+- `warehouse` The folder location where all of the `repositories` will be cloned into and where forklift will scan for pallets to lift.
+- `repositories` A list of github repositories in the `<owner>/<name>` format that will be cloned/updated into the `warehouse` folder.
+- `stagingDestination` The folder location where forklift creates and manages data before being copied to `copyDestinations`. This allows data in "production" to not be affected while forklift is running and if there are any issues. Data will only be copied if all crates are processed successfully. This is a helper method for creating crates. Usage would be from within a Pallet: `os.path.join(self.staging_rack, 'the.gdb')`
+- `copyDestinations` An array of folder locations that forklift will  copy data to. This is the "production" drop off location. The data is defined in `Pallet.copy_data` and is copied upon successful processing of the pallet.
 - `configuration` A configuration string (`Production`, `Staging`, or `Dev`) that is passed to `Pallet:build` to allow a pallet to use different settings based on how forklift is being run. Defaults to `Production`.
+- `sendEmails` A boolean value that determines whether or not to send forklift summary report emails after each lift.
+- `notify` An array of emails that will be sent the summary report each time `forklift lift` is run.
 
 Any of these properties can be set via the `config set` command like so:
 ```
 forklift config set --key sendEmails --value False
 ```
 If the property is a list then the value is appended to the existing list.
 
-### Install to First Successful Run
+## Install to First Successful Run
+
 1. `pip install .\` from the directory containing `setup.py`.
 1. `forklift config init`
 1. `forklift config set --key copyDestinations --value c:\\MapData` - This is where you want your output placed.
+1. `forklift config set --key stagingDestinations --value c:\\staging` - This is where your data is staged for copy.
```

Comment:
This is optional, right? There is a default value. I would think that we should remove it to provide the shortest path to a successful first run.